openmdao/docs/jb_utils.py

```diff
@@ -0,0 +1,75 @@
+import importlib
+import inspect
+from IPython.display import display, HTML, Code
+
+
+def get_object_from_reference(reference):
```

Comment:
could use a docstring here, and on the module itself 
(it's not  really clear what `jb` stands for... I assume jupyter book, but elsewhere we use `notebook`.. maybe `notebook_utils` is better?)

Reply:
I'll add that docstring. I felt that since these function should only be used for docs, jb_utils is more appropriate than `notebook_utils`.